src/images/opengl/castleglimages_sprite.inc

```diff
@@ -498,6 +508,12 @@ begin
   FImage.Rotation := AValue;
 end;
 
+procedure TSprite.SetColor(AValue: TCastleColor);
+begin
+  FImage.Color := AValue;
+end;
+
+
```

Comment:
An extra newline -- please remove :)

Reply:
Sorry, I noted this one, but thought about fixing it in some "refactoring" commit later (not to spawn a whole commit for just one line deletion) :) Done.